src/components/Request/FormFields/Incentives.tsx

```diff
@@ -0,0 +1,47 @@
+import { Label, Radio, RadioGroup, Text } from "@fluentui/react-components";
+import { Controller } from "react-hook-form";
+import { FormField } from "components/Request/NewRequestForm";
+import "components/Request/Request.css";
+import { RadioButtonFilled } from "@fluentui/react-icons";
+
+const Incentives = ({ name, form }: FormField) => {
+  return (
+    <div className="requestFieldContainer">
+      <Label
+        htmlFor={name + "Id"}
+        size="small"
+        weight="semibold"
+        className="requestFieldLabel"
+        required
+      >
+        <RadioButtonFilled className="requestFieldIcon" />
+        Recuitment, Retention, and Relocations incentives are avialable
```

Comment:
I was just going to suggest Relocation instead of Relocations, and GitHub suggestion spell check caught the other 2 :)
```suggestion
        Recruitment, Retention, and Relocation incentives are available
```